src/components/include/media_manager/media_manager.h

```diff
@@ -60,6 +60,8 @@ class MediaManager {
    */
   virtual const MediaManagerSettings& settings() const = 0;
 
+  virtual uint32_t DataSizeToMilliseconds(uint64_t data_size) const = 0;
```

Comment:
Might be good to add a function brief  for this function

Reply:
Added in https://github.com/smartdevicelink/sdl_core/pull/3057/commits/c61c05654ee2d4919b03e0e539eb86ba030f3aba